src/ProjectTemplates/test/WorkerTemplateTest.cs

```diff
@@ -25,6 +25,7 @@ public WorkerTemplateTest(ProjectFactoryFixture projectFactory, ITestOutputHelpe
         [OSSkipCondition(OperatingSystems.Linux, SkipReason = "https://github.com/dotnet/sdk/issues/12831")]
         [InlineData("C#")]
         [InlineData("F#")]
+        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/25404")]
```

Comment:
This causes a long running test but the failing build didn't timeout and instead failed so I'm quarantining this instead of disabling outright.